### ISSUE TITLE
Difficulty: Platform spawning

### ISSUE THREAD

**Issue by thomasrsbhs** [thomasrsbhs/ProgrammingL3#5]
* Easy, medium, hard - lives & number of totalPlatforms changes

**Comment by thomasrsbhs** [comment]
Medium and Hard difficulty was not being received and lives and totalPlatform values were running the default easy values. (Lives at 3 and totalPlatforms = 6 etc.)

This was due to incomplete difficulty if statements @ mainloop  for null cases:

if (difficulty != 'easy', 'medium', 'hard') { //runs the false statement so that unrecognised text is rejected 
      difficultyPass == false;    // was added and solved the issue allowing for difficulty change and new lives and totalPlatforms values to run

if (difficulty.toLowerCase() == 'easy' || difficulty.toLowerCase() == 'medium' || difficulty.toLowerCase() ==
      'hard') {
      difficultyPass = true; // if statement is compatable with criteria then run as true and game will run
    }

    if (difficulty != null && difficulty != '') {
      difficultyPass = true;
    }


// also changed the platformCount var to totalPlatforms to be more consistent

**Comment by thomasrsbhs** [comment]
Added new if statements to difficutly string & mainloop which allowed user entering Medium or Hard difficulties to change live and totalPlatform values as intended. 

Improving the difficulty to change totalPlatforms running in game and lives was received through teacher and classmate feedback, originally speed of platforms and number of lives was all that was going to change with the difficulty. But instead I changed the totalPlatforms count and lives to scale with each difficulty to make the game more fun instead of brutally difficult.  For example:

If (difficulty == 'easy'){
   platYspeed = 5;
   platXspeed  =  5;
   lives = 3;

if (deaths >= 3 && lives <= 0) {
        gameRun = false;
    } // game end on deaths for easy

}  

instead having:
if (difficulty == 'medium') { //medium
    totalPlatforms = 5;
    lives = 4;

    if (deaths <= 4) {
      gameRun = true;
    } else {
      gameRun = false;
    }
  }
